planemo/bioconda_scripts/bioconductor_skeleton.py

```diff
@@ -0,0 +1,482 @@
+#!/usr/bin/env python
+
```

Comment:
I don't think we should ship this script with planemo. We maintaining it upstream and we would always need to sync, ideally this will be placed in conda directly.